docs/api/migration-guides/qiskit-algorithms-module.mdx

```diff
@@ -201,155 +201,129 @@ The legacy `qiskit.algorithms.minimum_eigen_solvers.VQE` class has now been spli
   [`qiskit.algorithms.minimum_eigensolvers.SamplingVQE`](../qiskit/qiskit.algorithms.minimum_eigensolvers.SamplingVQE)'s [`qiskit.algorithms.minimum_eigensolvers.SamplingVQEResult`](../qiskit/qiskit.algorithms.minimum_eigensolvers.SamplingVQEResult).
 </Admonition>
 
-```{eval-rst}
-.. dropdown:: VQE Example
-    :animate: fade-in-slide-down
+#### VQE Example
 
-    **[Legacy] Using Quantum Instance:**
+[Legacy] Using Quantum Instance:
```

Comment:
I removed bolding from the "Before" and "After" sections like this. It looked worse on OneDocs.